test/runner.server.lua

```diff
@@ -0,0 +1,93 @@
+--[[
+	This test runner is invoked in all the environments that we want to test our
+	library in.
+
+	We target Lemur, Roblox Studio, and Roblox-CLI.
+]]
+
+-- luacheck: globals __LEMUR__
+
+local isRobloxCli, ProcessService = pcall(game.GetService, game, "ProcessService")
+
+local function findUnitTests(container, foundTests)
+	foundTests = foundTests or {}
+
+	for _, child in ipairs(container:GetChildren()) do
+		if child:IsA("ModuleScript") then
+			table.insert(foundTests, child)
+		end
+
+		findUnitTests(child, foundTests)
+	end
+
+	return foundTests
+end
+
+local completed, result = xpcall(function()
+	local ReplicatedStorage = game:GetService("ReplicatedStorage")
+
+	local testModules = findUnitTests(ReplicatedStorage.TestEZTests)
+
+	local failureCount = 0
+	local successCount = 0
+	local errorMessages = {}
+
+	for _, testModule in ipairs(testModules) do
+		local tests = require(testModule)
+
+		print(string.format("%s", testModule.Name))
+
+		for testName, testFunction in pairs(tests) do
+			local success, message = pcall(testFunction)
+
+			if success then
+				print(string.format("  [PASS] %s", testName))
+				successCount = successCount + 1
+			else
+				print(string.format("  [FAIL] %s", testName))
+				failureCount = failureCount + 1
```

Comment:
Same as above; looks like all we need to know here is that something failed at all.

Reply:
Looks like most of this was adapted from the previous `spec.lua` file, but we dropped the print at the end with the number of successes and failures.